### ISSUE TITLE
Make SFX for Hovering Over A Menu Option

### ISSUE THREAD

**Issue by Pseudo-64** [Team-Awkward-Smile/Epilogue#415]
It is probably the most subtle sound of all the UI menus. But when the player hovers over a menu option, the stretching of flesh is heard.

Basically imagine holding someone's flesh in your hand, maybe you cut off someone's face. And then you stretch it like bacon strip. How would the muscle stretch, and how would it sound. It won't sound loud, but it would sound a little squishy, and the stretching of the meat would tighten its smaller particles in a hold to keep the flesh intact. 

A good example of this could be tightening of a rope. Like how the rope makes this stretching yet tightening noise when its stretch to its limits. Something like that, but with imagine, a string of flesh. 

Basically to put into a single sentence, "**Naked meat on a meat hook, barely holding on from falling.**"

**Comment by boliverse** [comment]
[UI_Select.zip](https://github.com/user-attachments/files/17339241/UI_Select.zip) @Hhk187 @vitormaduro

**Comment by Mediocre-At-Best** [comment]
[HoverOverButtons.zip](https://github.com/user-attachments/files/21376238/HoverOverButtons.zip) @Pseudo-64 @Hhk187
